### PR TITLE
Receipts streamline

### PR DESCRIPTION
**Pull request by WHYTEWYLL**
adding Retry logic

### CODE REVIEW

File: models/silver/retry/_pending_status.sql [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+)
+
+select 
+    block_number,
+    REPLACE(
+        concat_ws('', '0x', to_char(block_number, 'XXXXXXXX')),
+        ' ',
+        ''
+    ) AS block_number_hex,
+    tx_hash
```
Comment (forgxyz) [inline models/silver/retry/_pending_status.sql:30]: Compilation error, tx_hash is not in the subject CTE

**Review by forgxyz — CHANGES_REQUESTED** [review]
`_pending_status` model does not compile

File: models/silver/streamline/realtime/streamline__blocks_realtime.sql [hunk 2]
```diff
@@ -53,3 +53,14 @@ SELECT
     ) AS params
 FROM
     tbl
+UNION
+SELECT
+    block_number,
```
Comment (forgxyz) [inline models/silver/streamline/realtime/streamline__blocks_realtime.sql:58]: We need a DISTINCT here. As TX_HASH was added to the ephemeral model, we'll likely get multiple TXs per block

File: models/silver/streamline/realtime/streamline__transactions_realtime.sql [hunk 3]
```diff
@@ -52,4 +52,15 @@ SELECT
         'true'
     ) AS params
 FROM
-    tbl
\ No newline at end of file
+    tbl
+UNION
+SELECT
+    block_number,
```
Comment (forgxyz) [inline models/silver/streamline/realtime/streamline__transactions_realtime.sql:58]: Looks like TXs are also requested at the block level, so will need a DISTINCT here as well.

**Review by forgxyz — CHANGES_REQUESTED** [review]
2 possible solutions
1. Change the SELECT list in the streamline models where the request is block level to DISTINCT
2. Make 2 `retry` models, one that just gets block number while a second that gets tx hash

File: models/silver/streamline/realtime/streamline__blocks_realtime.sql (hunk 2 above)
Comment (WHYTEWYLL) [reply models/silver/streamline/realtime/streamline__blocks_realtime.sql:58]: Doesnt UNION take care of the duplicates?

Comment (WHYTEWYLL) [reply models/silver/streamline/realtime/streamline__blocks_realtime.sql:58]: On the silver side, we also have the qualify on the model taking care of it. Isnt?

**Pull request merged**